packages/falso/src/lib/food.ts

```diff
@@ -0,0 +1,72 @@
+import { FakeOptions, fake, getRandomInRange } from './core/core';
+import { data } from './food.json';
+import { rand } from './rand';
+
+// TODO: Add more Foods to JSON
+
+export type CuisineOrigin =
+  | 'china'
+  | 'italy'
+  | 'india'
+  | 'mexico'
+  | 'japan'
+  | 'korea'
+  | 'france'
+  | 'lebanon'
+  | 'thailand'
+  | 'greece'
+  | 'turkey';
+
+export interface FoodOptions extends FakeOptions {
+  origin?: CuisineOrigin;
+}
+
+/**
+ * Generate a random food.
+ *
+ * @category Food
+ *
+ * @example
+ *
+ * randFood()
+ *
+ * @example
+ *
+ * randFood({ origin: 'italy' })
+ *
+ * @example
+ *
+ * randFood({ length: 10 })
+ *
+ */
+export function randFood<Options extends FoodOptions>(options?: Options) {
+  const foodData: { [origin: string]: string[] } = data;
```

Comment:
Why do we need this variable?

Reply:
For some reason the type isn't inferred so we get an error when we try to select an origin by key

```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ china: string[]; italy: string[]; india: string[]; mexico: string[]; japan: string[]; korea: string[]; france: string[]; lebanon: string[]; thailand: string[]; greece: string[]; turkey: string[]; }'.
      No index signature with a parameter of type 'string' was found on type '{ china: string[]; italy: string[]; india: string[]; mexico: string[]; japan: string[]; korea: string[]; france: string[]; lebanon: string[]; thailand: string[]; greece: string[]; turkey: string[]; }'.
```

I figured that i'd look into other solutions one I had the basic concept working